praw/models/helpers.py

```diff
@@ -4,13 +4,127 @@
 
 from ..const import API_PATH
 from .base import PRAWBase
+from .reddit.draft import Draft
 from .reddit.live import LiveThread
 from .reddit.multi import Multireddit, Subreddit
 
 if TYPE_CHECKING:  # pragma: no cover
     import praw
 
 
+class DraftHelper(PRAWBase):
+    """Provide a set of functions to interact with Drafts."""
+
+    def __call__(
+        self, draft_id: Optional[str] = None
```

Comment:
Should we make this a mandatory keyword argument?

Reply:
No, because calling this without an id will return all of them. If you have an id, you can only return that one. This is similar to how removal reasons are implemented.